blog/_posts/2016-10-6-start-nonprofit.md

```diff
@@ -0,0 +1,246 @@
+path was not entirely clear, making it difficult to see the plan from start to
+finish.
+
+So, the other day, co-founder Matt Bailey and I sat down and tried to put the
+pieces together into a coherent plan. Here's the process, as we understand it so
+far.
+
+<!--more-->
+
+Because organizations are primarily regulated at the state level, we began
+there. At first, the process appeared to involve three separate DC agencies:
+
+- the Department of Consumer and Regulatory Affairs (DCRA),
+- the Office of the Deputy Mayor for Planning and Economic Development (DMPED), and
+- the Office of Tax and Revenue (OTR)
+
+Both DCRA and DMPED have pages about creating a nonprofit. [The DCRA
+page](http://dcra.dc.gov/service/register-and-license-non-profit-organization)
+covers the main steps, while [the DMPED page](http://dmped.dc.gov/node/814892)
+covers over-arching questions and tasks, some of which overlap with DCRA steps.
+At first, this caused some confusion, as it was unclear how the pages related to
+each other.
+
+However, since a registrant files the form with DCRA, that seems like the
+primary one. Additionally, we figured out that you don't complete all of the
+state steps before completing the federal steps. Instead, you go back and forth
+between the two. The Free Law Project (FLP) went through the process a couple years
+ago in California and [wrote up their
+steps](http://michaeljaylissner.com/posts/2014/10/14/becoming-a-non-profit/),
+which helped make the flow a little more clear.
+
+## Brief detour: the Unified Registration statement
+
+At the bottom of the DCRA guide, they say:
+
+> Many customers have asked about using the Unified Registration Statement (URS). We do accept the URS, however you will still need to apply for a Charitable Solicitation BBL if you directly or indirectly solicit for any contribution in the District of Columbia.
+
+We had no idea what that meant, so we looked up the URS. It appears to be a form
+to [standardize nonprofit registration across
+states](http://multistatefiling.org/). However, while it's accepted by DC, it
+seems to require the same stuff, and that statement isn't exactly a glowing
+endorsement. Since the last update to the URS was in March 2014, we didn't think
+that this was a great idea.
+
+It seems like DC could pretty easily switch over to using the URS and fully
+support the standard, or otherwise should not introduce an additional wrinkle
+into the registration process.
+
+Anyway, combining the various resources that we've found, these seem to be the main steps:
+
+## Organize information and other prep work
+
+### Draft mission statement
+
+This is not specifically a required step, but it is a useful exercise and will
+come in handy as you fill out forms again and again. The mission statement
+should be short and to the point. You can use the [OpenGov
+Foundation's](https://github.com/opengovfoundation/legal-information/blob/master/bylaws.md#section-1-purposes)
+as an example.
+
```

Comment:
As I mentioned offline, I wouldn't recommend using our mission statement from OpenGov as a good example for this type of organization, as we've seen increasing resistance from the IRS to granting 501(c)(3) exemptions for open source software & technology.  (The argument goes that anyone can use the software for-profit as well, so it's not 100% in the public good.) If we had it to do over again, I'd downplay that part and focus on the public engagement piece instead.